https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface/pull/39

From 032a3e38c62b84ec56fad70d24cf82ee65d8c80b Mon Sep 17 00:00:00 2001
From: Craig Andrews <candrews@integralblue.com>
Date: Mon, 22 Jul 2019 17:06:16 -0400
Subject: [PATCH] Install libhsakmt.pc to the standard location

pkgconfig files should be installed to /usr/share/pkgconfig/, not /usr/libhsakmt/
---
 hsakmt-dev.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hsakmt-dev.txt b/hsakmt-dev.txt
index eb0d3e2..195cc6c 100644
--- a/hsakmt-dev.txt
+++ b/hsakmt-dev.txt
@@ -43,7 +43,7 @@ set ( BUILD_VERSION_PATCH @BUILD_VERSION_PATCH@ )
 set ( CMAKE_VERBOSE_MAKEFILE on )
 
 ## Set the install targets
-install ( FILES libhsakmt.pc DESTINATION libhsakmt )
+install ( FILES libhsakmt.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/share/pkgconfig )
 install ( DIRECTORY ${SOURCE_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${HSAKMT_COMPONENT} )
 
 
